lib/ssl/src/ssl.erl

```diff
@@ -1449,22 +1451,70 @@ update_keys(#sslsocket{pid = [Pid, Sender |_]}, Type0) when is_pid(Pid) andalso
 update_keys(_, Type) ->
     {error, {illegal_parameter, Type}}.
 
+%%--------------------------------------------------------------------
+-spec exporters(SslSocket, Labels, Contexts, WantedLengths) ->
+                 {ok, Exporters} | {error, reason()} when
+      SslSocket :: sslsocket(),
+      Labels :: [binary()],
+      Contexts :: [binary() | no_context],
+      WantedLengths :: [non_neg_integer()],
+      Exporters :: [binary()].
+%%--------------------------------------------------------------------
+exporters(#sslsocket{pid = [Pid|_]}, Labels, Contexts, WantedLengths) when is_pid(Pid) ->
+    ssl_gen_statem:call(Pid, {exporters, Labels, Contexts, WantedLengths, true});
+exporters(#sslsocket{pid = {_Listen, #config{}}}, _,_,_) ->
+    {error, enotconn}.
+
+%%--------------------------------------------------------------------
+-spec exporters(SslSocket, Labels, Contexts, WantedLengths, Last) ->
```

Comment:
Having only one label is only a special case of having several.  The RFC says that we should generate the inner secret for all labels at once then delete the master secret before finishing the calculation that can only be done by us. Also For the user to do the list comprehension would mean sending N messages back and forth instead of one. There is a version
of the function without the flag as argument that defaults the flag to true.

Reply:
Ok. I might have missed that RFC part. can you give me ref?

Reply:
It is the part of the RFC that I pasted into this PR 2 days ago!

Reply:
I remember reading it, but thought it was contributor comment.

Reply:
aah. Appendix [E.1.4]. fine. let it be lists then.

this is kind suggesting single arguments. it got me confused.
![image](https://github.com/erlang/otp/assets/4620572/9661884e-83f5-4f4e-90f5-41554d955981)